src/sophys/common/devices/ocean.py

```diff
@@ -30,16 +30,24 @@ class OceanOpticsSpectrometer(Device):
     boxcar = Component(EpicsSignal, "SetBoxcar", lazy=True)
 
     spectra = Component(EpicsSignalRO, "Spectra", lazy=True)
```

Comment:
Why are those other signals hinted, and this one isn't?

Reply:
I only added the hinted kind to the attributes that are seen as important in the XANES scan from the CARNAUBA beamline, I don't know if this one is also important in their point of view.

Reply:
I'm just thinking in terms of general usage. If these particular signals are important for Carnaúba, would it perhaps make more sense to subclass this device there and configured these signals as hinted in its `__init__`?